packages/app-project/src/screens/ProjectAboutPage/components/AboutNavLink.js

```diff
@@ -23,12 +23,12 @@ const AboutNavLink = ({ router, link }) => {
 
   return (
     <Box
-      background={isCurrentPage ? 'accent-1' : 'neutral-6'}
+      background={isCurrentPage ? 'accent-1' : { light: 'neutral-6', dark: '' }}
       pad={{ horizontal: '20px', vertical: '5px' }}
     >
       <NavLink
         link={link}
-        color={isCurrentPage ? 'neutral-1' : 'dark-5'}
+        color={isCurrentPage ? 'neutral-1' : { dark: 'neutral-6', light: 'dark-5' }}
```

Comment:
I just realized I missed your question about this from my last review. I ran a lighthouse audit on this and interestingly it did not flag this as being an issue. However, storybook's accessibility auditor definitely did when these two colors were used together like this in the survey task. So I think to be safe, maybe we should just stay consistent with the original two set colors and not change this based on being on the current page?

```suggestion
        color={{ dark: 'neutral-6', light: 'dark-5' }}
```

As an aside, I realized also that we don't have stories for theses components, so this is not a super high priority, but whenever you have extra time, I think it'd be helpful to add a few key stories for this part of the app, maybe one for the side bar, one for the team list, and one for the main markdown content? We can just open an issue to track this for now.

Reply:
Sounds good! I've opened #2317 to add stories for those components.